components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -6982,7 +6981,7 @@ public static void savePolicyPartialMappings(Integer policyGroupId,
 	 * @throws AppManagementException if any an error found while saving data to DB
 	 */
 	public static void deletePolicyPartialMappings(Integer policyGroupId, Connection conn)
-			throws AppManagementException {
+			throws SQLException {
```

Comment:
This is a private method. (Access modifier has been changed later as 'private' with a subsequent commit)